### PR TITLE
Rchain 3502: split blessed contracts into lang and rchain

### PR DESCRIPTION
**Pull request by odeac**
## Overview
All the blessed contracts can be referred either via rho:lang or rho:rchain which is confusing for the users.

This PR splits them and allow only one location for these blessed contracts

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3502

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/test/scala/coop/rchain/casper/genesis/contracts/RegistrySpec.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package coop.rchain.casper.genesis.contracts class RegistrySpec {
+
+}
```
Comment (KentShikama) [inline casper/src/test/scala/coop/rchain/casper/genesis/contracts/RegistrySpec.scala]: What does this do?

Comment (odeac) [reply casper/src/test/scala/coop/rchain/casper/genesis/contracts/RegistrySpec.scala]: It's a rebase error but now it's fixed

**Pull request merged**